frontend/src/components/ShootWorkers/GWorkerInputGeneric.vue

```diff
@@ -320,40 +369,23 @@ export default {
         ? []
         : this.initialZones
     },
-    machineTypes () {
-      return this.machineTypesByCloudProfileRefAndRegionAndArchitecture({
-        cloudProfileRef: this.cloudProfileRef,
-        region: this.region,
-        architecture: this.machineArchitecture,
-      })
-    },
     volumeInCloudProfile () {
       return !isEmpty(this.volumeTypes)
     },
-    selectedMachineType () {
-      return find(this.machineTypes, ['name', this.worker.machine.type])
-    },
-    selectedVolumeType () {
-      return find(this.volumeTypes, ['name', this.worker.volume?.type])
+    filteredMachineImages () {
+      return filter(this.machineImages, ({ isExpired, architectures }) => !isExpired && includes(architectures, this.machineArchitecture))
     },
-    machineImages () {
-      const machineImages = this.machineImagesByCloudProfileRef(this.cloudProfileRef)
-      return filter(machineImages, ({ isExpired, architectures }) => !isExpired && includes(architectures, this.machineArchitecture))
-    },
-    minimumVolumeSize () {
-      const minimumVolumeSize = convertToGi(this.minimumVolumeSizeByMachineTypeAndVolumeType({
-        machineType: this.selectedMachineType,
-        volumeType: this.selectedVolumeType,
-      }))
+    useMinimumVolumeSize () {
```

Comment:
```suggestion
    minimumVolumeSize () {
```
avoid `use` syntax for computed property. We only have `useXYZ` for functions that initialize and return a composable. Also, this specific computed prop is inside of a component.